RiotSwiftUI/Modules/Room/PollHistory/View/PollHistory.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct PollHistory: View {
+    @Environment(\.theme) private var theme
+    
+    @ObservedObject var viewModel: PollHistoryViewModel.Context
+    
+    var bindings: PollHistoryViewBindings {
+        viewModel.viewState.bindings
+    }
```

Comment:
Looks unused to me :)

Reply:
Leftover! 😅
https://github.com/vector-im/element-ios/pull/7267/commits/8b692295b183981973b2e32afb5a7efbe57e3391